packages/frontend-passthrough-headers/project.json

```diff
@@ -1,28 +0,0 @@
-{
-  "name": "frontend-passthrough-headers",
-  "$schema": "../../node_modules/nx/schemas/project-schema.json",
-  "sourceRoot": "packages/frontend-passthrough-headers/src",
-  "projectType": "library",
-  "targets": {
-    "lint": {
-      "executor": "@nrwl/linter:eslint",
-      "options": {
-        "lintFilePatterns": [
-          "packages/frontend-passthrough-headers/**/*.{js,ts,json}"
-        ]
-      }
-    },
-    "format": {
```

Comment:
Do we need to keep this file in for the linting commands or will we just amend when we relocate the passthrough headers

Reply:
Yeah my thinking here is that this doesn't look too different from the other packages, so we can fairly easily recreate these when we move the packages in.

Reply:
Makes sense, if we have once version we can follow suit but will need to add it as a requirement